entity-framework/core/modeling/query-types.md

```diff
@@ -64,18 +60,18 @@ Next, we define a class to hold the result from the database view:
 
 [!code-csharp[Main](../../../samples/core/QueryTypes/Program.cs#QueryType)]
 
-Next, we configure the query type in _OnModelCreating_ using the `modelBuilder.Query<T>` API.
-We use standard fluent configuration APIs to configure the mapping for the Query Type:
+Next, we configure the keyless entity type in _OnModelCreating_ using the `HasNoKey` API.
+We use standard fluent configuration API to configure the mapping for the keyless entity type:
 
 [!code-csharp[Main](../../../samples/core/QueryTypes/Program.cs#Configuration)]
```

Comment:
Should update this code to be not so fluent.

Reply:
Could you elaborate? 😄

Reply:
more like
```c#
modelBuilder.Entity<(b => 
{
    b.ToView("");
    b.Property().HasColumnName();
});
```

Reply:
I think it might also be good to update it not to use deprecated API :trollface: